test/integration/mount_start_test.go

```diff
@@ -87,13 +93,23 @@ func TestMountStart(t *testing.T) {
 }
 
 // validateStartWithMount starts a cluster with mount enabled
-func validateStartWithMount(ctx context.Context, t *testing.T, profile string) {
+func validateStartWithMount(ctx context.Context, t *testing.T, profile string, hostPath string) {
```

Comment:
not a problem in this PR but wanted to let you know 
functions that start with vadliate...in integraiton test will be auto-published to the site with their comment, to show what things are being tests in minikube
https://minikube.sigs.k8s.io/docs/contrib/tests/